### PR TITLE
[ADD] ebay: new doc for account deletion notifications

### PR DESCRIPTION
**Pull request by odoo-dama**
task-2504306

Enterprise PR: https://github.com/odoo/enterprise/pull/20265

### CODE REVIEW

File: content/applications/sales/sales/ebay_connector/deletion_notifications.rst
Comment (Feyensv) [inline content/applications/sales/sales/ebay_connector/deletion_notifications.rst:7]:
```suggestion
reception of the request and take further action if necessary.
```

File: content/applications/sales/sales/ebay_connector/subscribe.rst
Comment (odoo-dama) [reply content/applications/sales/sales/ebay_connector/subscribe.rst]: Since we need the doc today, let's first release this version and adapt after as needed.

**Review by jcs-odoo — APPROVED** [review]

File: content/applications/sales/sales/ebay_connector/deletion_notifications.rst
Comment (AntoineVDV) [inline content/applications/sales/sales/ebay_connector/deletion_notifications.rst:24]: "Saas" for the average Odoo user is gibberish. Maybe "Odoo Online or Odoo.sh"?

File: content/applications/sales/sales/ebay_connector/deletion_notifications.rst
Comment (AntoineVDV) [inline content/applications/sales/sales/ebay_connector/deletion_notifications.rst:15]: We should make a relative reference to the related step below to let the user know that this part is covered.

File: content/applications/sales/sales/ebay_connector/deletion_notifications.rst
Comment (AntoineVDV) [inline content/applications/sales/sales/ebay_connector/deletion_notifications.rst:9]:
```suggestion
Odoo has a notification endpoint to receive those notifications, confirm the reception of the request, and handle the first set of actions
```
Otherwise, the previous paragraph leads to believe the user has to confirm something.

File: content/applications/sales/sales/ebay_connector/deletion_notifications.rst
Comment (AntoineVDV) [inline content/applications/sales/sales/ebay_connector/deletion_notifications.rst:12]:
```suggestion
.. important::
```

File: content/applications/sales/sales/ebay_connector/deletion_notifications.rst
Comment (AntoineVDV) [inline content/applications/sales/sales/ebay_connector/deletion_notifications.rst:36]:
```suggestion
Subscribe to account deletion notifications
```

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

**Comment by jcs-odoo** [comment]
@odoo/doc-review 
is it good to go? I know @odoo-dama wanted it online asap.
Thanks :)

**Comment by AntoineVDV** [comment]
We must wait until Monday to merge the PR.

**Review by AntoineVDV — APPROVED** [review]
Waiting for the enterprise PR to be merged.

@odoo-dama @jcs-odoo FYI I merged the new page into the page `setup.rst` since the subscription to account deletion notifications is now a mandatory step of the integration setup. I also re-targeted the PR to merge the doc in 12.0 rather than 14.0 since the enterprise PR will land in 12.0 too.

**Comment by AntoineVDV** [comment]
@odoo-dama I just noticed that we didn't mention the module to install. It's now done [here](https://github.com/odoo/documentation/pull/1115/files#diff-b4a2e2331b7351578675be7bd9d99737b3f98fc776a43f2cb4b10b533620a1ebR55-R97).
cc @jcs-odoo At some point we should extract these instructions in a separate page ;)

**Comment by AntoineVDV** [comment]
@robodoo r+

**Pull request closed without merging**
